git-changebar/src/gcb-plugin.c

```diff
@@ -45,6 +45,10 @@
                     new_buffer, new_len, new_as_path, options, \
                     file_cb, hunk_cb, line_cb, payload)
 #endif
+#if LIBGIT2_VER_MAJOR == 0 && LIBGIT2_VER_MINOR < 28
+#define git_buf_dispose(x) git_buf_free(x)
+#define git_error_last() giterr_last()
```

Comment:
there's also no need to include arguments as they are identical, it would be enough to do
```c
# define git_buf_dispose  git_buf_free
# define git_error_last   giterr_last
```

Reply:
Fixed, thanks for the review